python/paddle/distributed/auto_parallel/api.py

```diff
@@ -321,6 +324,90 @@ def __call__(self, *args):
             else:
                 return None
 
+    def state_dict(self, mode="all"):
+        """
+        Args:
+            mode(str, optional): Source of the obtained parameters and buffers.
+                'opt' :  The return value only contains the variable in the optimizer.
+                'param' : The return value only contains the variable in the network, not the variable in the optimizer.
+                'all' : The return value contains the variable in the network and optimizer.
+                Default: 'all'
+        """
```

Comment:
"Source of the obtained parameters and buffers" is a little bit confused for me. Maybe replace with "The type of returned parameters and buffers" ?